frontend/src/app/shared/components/editor/components/ckeditor-augmented-textarea/ckeditor-augmented-textarea.component.ts

```diff
@@ -184,6 +184,11 @@ export class CkeditorAugmentedTextareaComponent extends UntilDestroyedMixin impl
       });
   }
 
+  public editorFocused():void {
+    window.OpenProject.pageState = 'edited';
```

Comment:
When I just focus the comment field and then switch to another WP, I still see the error message. Ideally, I would only see that when the content is really edited.